src/platforms/python/integrations/grpc/index.mdx

```diff
@@ -50,24 +57,40 @@ server = grpc.server(
 import grpc
 
 import sentry_sdk
-from sentry_sdk.integrations.grpc.client import ClientInterceptor
+from sentry_sdk.integrations.grpc import GRPCIntegration
 
 sentry_sdk.init(
     dsn="___PUBLIC_DSN___",
     enable_tracing=True,
+    integrations=[
+        GRPCIntegration(),
+    ],
 )
 
 ...
 
+# this works with synchronous clients:
 with grpc.insecure_channel("example.com:12345") as channel:
-    channel = grpc.intercept_channel(channel, *[ClientInterceptor()])
+    ...
+
+# ... and asynchronous clients:
+async with grpc.aio.insecure_channel("example.com:12345") as channel:
+    ...
 ```
 
+<Note>
+
+**Note:**
+
+In older versions of this integration you had to add the interceptors by hand. Since Python SDK version X.X.X you do not need to add any interceptors by hand but only add the GRPCIntegration as described above.
```

Comment:
Don't forget to add this version # before merging!

Reply:
thanks for reminding!